addons/aircraft/XEH_PREP.hpp

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I like those for included files personally. Makes it easier to read the decompiled code later sometimes.